cypress.config.ts

```diff
@@ -43,6 +44,7 @@ export default defineConfig({
       openMode: 0,
     },
     experimentalMemoryManagement: true,
+    video: true,
```

Comment:
please remove it as it significantly slows down the test execution. It can be used only for debugging.

Reply:
It will be removed in the end.